src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -59,16 +64,27 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $container = $this->getApplication()->getKernel()->getContainer();
         $serviceId = $input->getArgument('name');
+        $format = $input->getOption('dump_format');
```

Comment:
I like to declare var very near where I used them. Could you move it line 80?

Reply:
moved